ocs_ci/templates/ocp-deployment/install-config-rhv-ipi.yaml.j2

```diff
@@ -7,8 +7,8 @@ compute:
   platform:
     ovirt:
       cpu:
-        cores: {{ worker_num_cpus | default(4) }}
-        sockets: 1
+        cores: {{ worker_num_cores | default(4) }}
+        sockets: {{ worker_num_sockets | default(1) }}
```

Comment:
Doesn't this change the default for other implementation and set it to 1? I think better to set this in RHV only block.

Reply:
This template file `install-config-rhv-ipi.yaml.j2` and other conf files in this PR are specific to RHV platform.
So it shouldn't affect other platform implementations.